src/palace/manager/api/circulation.py

```diff
@@ -822,8 +822,8 @@ def delete_loans_or_holds(self, loans_or_holds: Iterable[LoanOrHoldT]) -> None:
                 # This was just created, we shouldn't delete it.
                 continue
             self.log.info(
-                "Deleting %s (%r) for patron (%s)"
-                % (item.__class__.__name__, item, item.patron.authorization_identifier)
+                f"Deleting {item.__class__.__name__} ({item.id}) on license pool (id: {item.license_pool_id}) "
+                f"for patron ({item.patron.authorization_identifier})"
```

Comment:
This is a small change to logging that I made while troubleshooting this. Previously were we printing the `__repr__` for the `Loan` or `Hold` object. Since neither of them defines `__repr__`, we were just getting something like `Deleting Loan (<palace.manager.sqlalchemy.model.patron.Loan object at 0x7fa324d5e9e0>) for patron`, which is pretty useless for debugging. The updated log line prints the LP ID, so its a bit easier to track down what is happening.